scripts/tools/zap/tests/outputs/all-clusters-app/app-templates/gen_config.h

```diff
@@ -586,3 +586,264 @@
 #define ZCL_USING_FAULT_INJECTION_CLUSTER_SERVER
 #define MATTER_DM_PLUGIN_FAULT_INJECTION_SERVER
 #define MATTER_DM_PLUGIN_FAULT_INJECTION
+
+/**** Cluster Commands Flag ****/
+//  ActivatedCarbonFilterMonitoring Cluster Commands
+#define ACTIVATED_CARBON_FILTER_MONITORING_ENABLE_RESET_CONDITION 1
```

Comment:
nit: maybe a `_COMMAND` suffix on all of these would make it clearer on what is being enabled.